sites/main-site/src/layouts/MarkdownTocLayout.astro

```diff
@@ -166,6 +180,33 @@ if (showRightSidebar) {
                             </a>
                         )}
                         <SidebarNav items={sections} />
+                        {showTocInLeftSidebar && (
+                            <div class="toc mt-3">
```

Comment:
for navigation consistency ToCs should always be on the right. you can have a right-hand sidebar with some general navigation, but that is a different element to this one.